### DISCUSSION TITLE
ConditionCommands

### DISCUSSION THREAD

**Discussion by GGreenix** [mjansen4857/pathplanner#913] [Ideas]
As simple as it sounds, I've found myself couple of times in use of it, for example this year if a note was taken you could go to another note instead of continue as usual

**Comment by GGreenix** [comment]
I can try and work on that

**Comment by mjansen4857** [comment]
Conditional commands is not something I want to add to PathPlanner at this point, as it is very complex to implement in an easy to use way, and doesn't really end up being much easier than doing it manually in code anyways. The auto building functionality in PP is really just meant for simple stuff, any teams wanting more complex functionality than what is already there should just create the auto command in code.
